netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
+def static addCopyCoreClrAndRunTestSteps(def job, String os, String osGroup, String fullNativeCompBuildJobName, String fullCoreFXBuildJobName, String configurationGroup, String coreClrConfigurationGroup, boolean isOuterLoop, boolean useServerGC) {
+    job.with {
+        steps {
+            // CoreCLR
+            copyArtifacts("dotnet_coreclr/master/${coreClrConfigurationGroup.toLowerCase()}_${os.toLowerCase()}") {
+                excludePatterns('**/testResults.xml', '**/*.ni.dll')
+                buildSelector {
+                    latestSuccessful(true)
+                }
+            }
+
+            // MSCorlib
+            copyArtifacts("dotnet_coreclr/master/${coreClrConfigurationGroup.toLowerCase()}_windows_nt") {
+                includePatterns("bin/Product/${osGroup}*/**")
+                excludePatterns('**/testResults.xml', '**/*.ni.dll')
+                buildSelector {
+                    latestSuccessful(true)
+                }
+            }
+
+            // Native components
+            copyArtifacts(fullNativeCompBuildJobName) {
+                includePatterns("bin/**")
+                buildSelector {
+                    buildNumber('\${COREFX_NATIVECOMP_BUILD}')
+                }
+            }
+
+            // The tests/corefx components
+            copyArtifacts(fullCoreFXBuildJobName) {
+                includePatterns('bin/build.pack')
+                buildSelector {
+                    buildNumber('\${COREFX_BUILD}')
+                }
+            }
+
+            // Unpack the build data
+            shell("unpacker ./bin/build.pack ./bin")
+            // Export the LTTNG environment variable and then run the tests
+            shell("""export LTTNG_HOME=/home/dotnet-bot
```

Comment:
Just curious, is this LTTNG_HOME thing still needed?  I'm wondering if the underlying issue has been fixed.  Regardless, it's fine for now.